### DISCUSSION TITLE
Help with Manual Installation: Create Web Application section

### DISCUSSION THREAD

**Discussion by ionicbond** [RaspAP/raspap-webgui#992] [Q&A]
I'm having a problem understanding what this step means under "Create Web Application" section found in the beginning of the manual Installation instructions:

"Copy an extra lighttpd config file to support application routing. This step requires some text substitutions to support user changes to lighttpd's server.document-root setting:"

It's not clear what this extra lighttpd  config file is supposed to be called and where it is supposed to go.  Can someone help me with this?

**Comment by billz** [comment]
The code block following “Copy an extra lighttpd config...” does the work of modifying and installing `50-raspap-router.conf` for you. You can either copy-paste this block and execute it all at once or enter it line-by-line. The end result is the same.

**Reply by debbborah** [reply]
Sorry if I ask a stupid question, but in the sixth line 'awk "{gsub(\"/REPLACE_ME\",\"$HTROOT\")}1" $CONFSRC > /tmp/50-raspap-router.conf' I therefore suspect that I need to replace 'REOLACE_ME', but with what?

**Comment by billz** [comment]
The placeholder value `REPLACE_ME` is replaced by the `awk` command with the value contained in `$HTROOT`. 
As mentioned above, you can simply copy-paste this code block and execute it in the shell.
